app/src/main/res/layout/inactive_header_item.xml

```diff
@@ -51,6 +52,7 @@
             app:layout_constraintBottom_toBottomOf="@id/inactive_title"
             app:layout_constraintStart_toEndOf="@id/inactive_title"
             app:layout_constraintTop_toTopOf="@id/inactive_title"
+            app:tint="@color/primary_text_normal_theme"
```

Comment:
`tab_tray_item_text_normal_theme` and `primary_text_normal_theme` surprisingly maps to the same colors. I think  we should just change `tab_tray_item_text_normal_theme` to `primary_text_normal_theme`, and I can examine removing `tab_tray_item_text_normal_theme` in a later followup with my color refactoring.